### DISCUSSION TITLE
[gelöst] gaszaehler wert wird falsch erkannt

### DISCUSSION THREAD

**Discussion by alero63** [jomjol/AI-on-the-edge-device#2653] [General] [answered]
Hi,

habe den Gaszähler soweit fertig. Lief bis dato gut. jetzt meint er eine deutlich erkennbare 6 als 8 zu erkennen. habe schon alles versucht. er will nicht von der 8 abrücken.

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/46607155/26954f18-5cfe-45bf-9ddf-7b10aaf2f55e)

**Comment by caco3** [comment, marked as answer]
deine ROIs sind zu gross! Schau dir bitte die Doku an!

**Reply by alero63** [reply]
hi, danke das wars.
